packages/core/src/ceramic.ts

```diff
@@ -454,10 +454,7 @@ class Ceramic implements CeramicApi {
    */
   async setDID(did: DID): Promise<void> {
     this.context.did = did
-    if (!did.authenticated) {
-      await this.did.authenticate()
-    }
-    this._logger.imp(`Now authenticated as DID ${did}`)
+    this._logger.imp(`Now using DID ${did}`)
```

Comment:
Not sure if this is really needed, but we should log when authenticating in the other places (then log `did.id` instead of `did`)

Reply:
okay will do, but it's kind of an a big change so I'm going to do it as a follow-up on top of this and https://github.com/ceramicnetwork/js-ceramic/pull/1197 to minimize rebase conflicts